subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/rule/ActivityRule.kt

```diff
@@ -7,8 +7,8 @@ import com.avito.android.test.Intents.stubEverything
 
 internal class ActivityRule<T : Activity>(aClass: Class<T>) : ActivityTestRule<T>(
     aClass,
-    /*initialTouchMode*/ true,
-    /*launchActivity*/ false
```

Comment:
It was convenient and only one option for Java code.
How about suppressing such cases?

Reply:
will try

Reply:
```
internal class ActivityRule<T : Activity>(aClass: Class<T>) : ActivityTestRule<T>(
    aClass,
    true, // initialTouchMode
    false // launchActivity
) {
```

this is ok i think